pkgs/tools/misc/hakuneko/default.nix

```diff
@@ -1,28 +1,89 @@
-{ stdenv, fetchurl, wxGTK30, openssl, curl }:
+{ atomEnv
+, autoPatchelfHook
+, dpkg
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+, udev
+, stdenv
+, wrapGAppsHook
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+
+  throwSystem = throw "Unsupported system: ${system}";
 
-stdenv.mkDerivation rec {
   pname = "hakuneko";
-  version = "1.4.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/hakuneko/hakuneko_${version}_src.tar.gz";
-    sha256 = "76a63fa05e91b082cb5a70a8abacef005354e99978ff8b1369f7aa0af7615d52";
+  version = "5.0.8";
+
+  meta = with stdenv.lib; {
+    description = "Manga & Anime Downloader";
+    homepage = "https://sourceforge.net/projects/hakuneko/";
+    license = licenses.unlicense;
+    maintainers = with maintainers; [
+      nloomans
+    ];
+    platforms = [
+      # Other platforms then linux_amd64 are supported by upstream, but I don't
+      # have the hardware to test those. PR's welcome!
+      "x86_64-linux"
+    ];
   };
 
-  preConfigure = ''
-    substituteInPlace ./configure \
-       --replace /bin/bash $shell
+  linux = stdenv.mkDerivation rec {
+    inherit pname version meta;
+
+    src = fetchurl {
+      url = "https://github.com/manga-download/hakuneko/releases/download/v${version}/hakuneko-desktop_${version}_linux_amd64.deb";
+      sha256 = "924df1d7a0ab54b918529165317e4428b423c9045548d1e36bd634914f7957f0";
+    };
+
+    desktopItem = makeDesktopItem {
```

Comment:
I would move this outside the derivation since it's not a standard attribute.

Reply:
Good point, fixed.